src/utils/prettyString.ts

```diff
@@ -0,0 +1,2 @@
+// This function removes uneccessary spaces from the input string
+export const prettyString = (input: string): string => input.trim().replace(/\s+/g, ' ');
```

Comment:
please do not create separate files for oneliners if they can be grupped thematically (and here they could become `stringUtils.ts` or `urlUtils.ts`)